javascript.html.markdown

```diff
@@ -603,31 +603,39 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement without '{ }' defined will be the return statement, hence the `return` keyword is not required. 
 
-const isEven = (number) => {
-    return number % 2 === 0;
-};
+const isOdd = number => number % 2 !== 0;
+
+// But, if '{ }' are used, then a `return` is required.
```

Comment:
Suggestion:
> If the body of an arrow function _is_ surrounded with braces (`{}`) then the function works as normal. If a return value is needed: use the `return` keyword.

Reply:
"required" is too strong of a word here in my opinion, it's only required if you need a return value.